_posts/2019-01-30-gcal-for-project-management.md

```diff
@@ -0,0 +1,79 @@
+
+**As an Account Manager, it can get hectic trying to manage all of this. [Google Calendar](https://www.google.com/calendar) can be a serious help.**
+
+<!--more-->
+
+#### Using Multiple Calendars
+
+In order to manage the different types of projects we work on at [Ombu Labs](https://www.ombulabs.com), it can be helpful to use multiple calendars.
+
+1. **General Admin/Personal**. This calendar includes our team's daily [scrum](https://www.scrum.org/resources/what-is-scrum) call, sprint retrospectives and occasional [donut calls](https://www.donut.com). Any general office-related or personal tasks could go here. Need to track your billable hours using a time-tracker? Need to create client invoices? Have a performance review coming up? These could all be scheduled in this calendar.
+
+2. **Ombu Labs Products & Open Source**. In this calendar, I have our team's weekly [OmbuShop](http://www.ombushop.com/) call, [FastRuby.io](https://fastruby.io/) [pair programming](https://www.ombulabs.com/blog/agile/pair-programming/joys-and-woes-of-pair-programming.html) sessions and some hours that are blocked off to work on open source projects.
+
+3. **Client Projects**. This calendar contains all events relating to our client projects. This includes demo calls, check-ins with clients and pair programming sessions.
+
+<img src="/blog/assets/images/multiple-calendars.png" alt="Multiple Calendars">
+
+#### Shared Calendars
+
+Shared calendars are a great way to get your team on board with a group of scheduled events. At [Ombu Labs](https://www.ombulabs.com), we use the shared calendars feature in our **Vacations Calendar** and our **Birthdays Calendar**.
+
+When a team member is going on vacation, they mark the days they will be away on the shared **Vacations Calendar**, so that their absence appears on the top of everyone else's calendar. This is preferable to the [_*Out of Office*_](https://gsuiteupdates.googleblog.com/2018/06/better-manage-your-work-and-personal_27.html ) feature, which only shows up on your individual calendar.
+
+The shared **Birthday Calendar** works similarly. If the calendar is enabled, team member birthdays will appear at the top of your calendar. Both of these shared calendars can be hidden by each team member to reduce clutter as well.
+
+<img src="/blog/assets/images/shared-calendar.png" alt="Shared Calendars">
+
+#### Managing Multiple Clients
+
+At [Ombu Labs](https://www.ombulabs.com), we typically work on two to three client projects at a time. As an Account Manager, it is important for me to dedicate time to working and managing each one of them.
```

Comment:
@ercohen14 This might change in the near future. With the current team size, we aim to work in 3 to 4 client projects. I don't know if it makes a difference to correct this.